src/main/kotlin/camp/nextstep/blackjack/card/CardDeck.kt

```diff
@@ -0,0 +1,24 @@
+package camp.nextstep.blackjack.card
+
+class CardDeck private constructor(private val _cards: MutableList<Card>) {
+
+    val cards get() = _cards.toList()
```

Comment:
[문서와 같이](https://kotlinlang.org/docs/properties.html#getters-and-setters) getter는 라인을 나눠서 작성해보면 어떨까요?

```suggestion
    val cards
        get() = _cards.toList()

```

Reply:
공유해주신 문서 아래에 
```
val area get() = this.width * this.height
```

타입 추론이 가능한 경우 한 라인에 작성하는 케이스도 있는 것 같은데요,
따로 정해진 컨벤션이 있는걸까요?

Reply:
코틀린의 컨벤션은 공식문서가 제일 정확하다고 생각해요!! :) 
타입추론이 가능할 경우 하나의 라인에 작성이 가능하다는 부분을 저도 이번에 처음알았네요..!! (감사합니다 ❤️)

저의 경우 전체 코드의 일관성을 유지시키기 위해 타입추론이 가능하더라도 타입을 명시하고, 라인을 나눠서 작성을하는데요!
현식님 지금처럼 작성하고 싶다고 하시면 그대로 작성해주셔도 무방할 것 같아요!